src/libraries/System.Private.CoreLib/src/System/Text/Latin1Encoding.cs

```diff
@@ -586,14 +583,13 @@ public override unsafe string GetString(byte[] bytes, int index, int count)
                 ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument.bytes, ExceptionResource.ArgumentOutOfRange_IndexCountBuffer);
             }
 
-            return string.Create(count, (encoding: this, bytes, index), static (chars, args) =>
+            string result = string.FastAllocateString(bytes.Length);
```

Comment:
```suggestion
            string result = string.FastAllocateString(count);
```
I do not think that `bytes.Length` is correct. This method does not convert the whole array. Let's see whether we have tests to catch this bug.

Reply:
Yup, we do:

```
Test System.Text.Tests.Latin1EncodingDecode.Decode has failed. Check the Test tab for details.
```